web/src/lib/components/admin-page/jobs/job-tile.svelte

```diff
@@ -119,12 +119,14 @@
         color="light-gray"
         on:click={() => dispatch('command', { command: JobCommand.Start, force: false })}
       >
-        <Icon path={mdiAlertCircle} size="36" /> DISABLED
+        <Icon path={mdiAlertCircle} size="36" />
+        {$t('admin.disabled').toUpperCase()}
```

Comment:
Should disabled be a general thing? I could imagine that being used in other non-admin places as well...?

Reply:
That already existed, but I will move it